profile/README.md

```diff
@@ -0,0 +1,10 @@
+Welcome to Pulumi open source! 💜 ☁️
+
+We are glad that you are here! We love and appreciate the Pulumi community.
+We use [Pulumi Slack](https://slack.pulumi.com/)
+and [Pulumi GitHub Discussions](https://github.com/pulumi/pulumi/discussions) for community conversations.
```

Comment:
Can we give guidance on which to use, when? Even if it's "Use whichever you feel comfortable with".